homeassistant/components/huawei_lte/sensor.py

```diff
@@ -267,6 +267,95 @@ class HuaweiSensorEntityDescription(SensorEntityDescription):
                 ),
                 entity_category=EntityCategory.DIAGNOSTIC,
             ),
+            "nrbler": HuaweiSensorEntityDescription(
+                key="nrbler",
+                translation_key="nrbler",
+                entity_category=EntityCategory.DIAGNOSTIC,
+            ),
+            "nrcqi0": HuaweiSensorEntityDescription(
+                key="nrcqi0",
+                translation_key="nrcqi0",
+                icon="mdi:speedometer",
+                entity_category=EntityCategory.DIAGNOSTIC,
+            ),
+            "nrcqi1": HuaweiSensorEntityDescription(
+                key="nrcqi1",
+                translation_key="nrcqi1",
+                icon="mdi:speedometer",
+            ),
+            "nrdlbandwidth": HuaweiSensorEntityDescription(
+                key="nrdlbandwidth",
+                translation_key="nrdlbandwidth",
+                # Could add icon_fn as in dlbandwidth, source for 5G thresholds?
```

Comment:
Rephrased some, suggestions welcome if more changes are wanted.

I don't think the comment should be removed, because it's a valuable TODO (which isn't marked as such because the HA linter config at least used to forbid TODO comments), and I don't think there's a better place than right there in the code where the thing to address is.